api-docs/api-reference/methods/post-save-instance-logs-version-accountid-instances-instanceid-action.rst

```diff
@@ -0,0 +1,125 @@
+
+.. _post-save-instance-logs-version-accountid-instances-instanceid-action:
+
+Save the DB logs for an instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+    POST /{version}/{accountId}/instances/{instanceId}/action
+
+Saves DB logs of the Instance.
```

Comment:
Saves DB logs of the Instance. -> Saves DB logs of the instance.  to be consistent with header in line 4.